javascript/ecmascript-2015/symbol/why-use-symbols-2.md

```diff
@@ -26,20 +26,20 @@ links:
 ---
 ## Content
 
-Probably the primary use of symbols is to introduce changes into EcmaScript. We can this in action with *iterators*.
+Probably the primary use of symbols is to introduce changes into EcmaScript. We can see this in action with `iterator`s.
```

Comment:
I think iterator can be left as italicized, it's not a word in the language itself.

Reply:
done